extensions/VisualEditor/wikia/modules/ve/dm/ve.dm.WikiaTransclusionModel.js

```diff
@@ -2,178 +2,180 @@
  * VisualEditor DataModel WikiaTransclusionModel class.
  * @license The MIT License (MIT); see LICENSE.txt
  */
+( function ( ve ) {
 
-ve.dm.WikiaTransclusionModel = function VeDmWikiaTransclusionModel() {
-	// Parent constructor
-	ve.dm.WikiaTransclusionModel.super.call( this );
-};
+	ve.dm.WikiaTransclusionModel = function VeDmWikiaTransclusionModel() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this function anonymous by removing its name: 'function() {...}'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)